index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
+                        <h1 class="contact">Contact me</h1>
+                            <p class="paragraph">If you have an application y ou are interested in developing,
+                                a feature that you need built or   a project that needs 
+                                coding. I’d love to help with it 
+                            </p>
+                        </section>
+                            <section>
+                                  <ul class="form-container">
+                                    <li class="input-name input-email-li">
+                                        <input  class="input-email" type="text" name="name" id="name" maxlength="30" placeholder="safari hamuli" required>
+                                    </li>
+                                    <li class="input-email-li">
+                                        <input class="input-email" type="email" name="email" id="email" placeholder="safari@gmail.com" required>
+                                        
+                                    </li>
+                                    <li class="input-email-li">
+                                        <textarea class="text-area" name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
+                                        placeholder="Enter your message here" required></textarea>
+                                    </li>
+                                    <li  class="submit-button input-email-li">
+                                        <button class="form-btn" type="submit">
+                                            Get in touch
+                                        </button><br>
+                                        <small></small>
```

Comment:
- [ ] Kindly remove this empty `small` tag to have a clean and professional codebase 👍